ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -104,8 +104,8 @@ open class DrawerController: UIViewController {
     }
 
     @objc public static func popoverBackground(fluentTheme: FluentTheme) -> UIColor {
-        return UIColor(dynamicColor: DynamicColor(light: fluentTheme.aliasTokens.colors[.background2].light,
-                                                  dark: fluentTheme.aliasTokens.colors[.background2].darkElevated))
+        return UIColor(dynamicColor: DynamicColor(light: fluentTheme.aliasTokens.colors[.background4].light,
```

Comment:
Why do we recreate the DynamicColor? Doesn't the alias token dynamic color already include the light / dark variants?

Reply:
Because the DynamicColor has DarkElevated as well, which can accidentally be hit and show the wrong background color when the popover/drawer is in Elevated mode (i.e., on ipad side-by-side)

Reply:
But isn't that the intended behavior, actually? Or should we only use darkElevated values if the spec explicitly calls for it?

Reply:
+1 on Mike's question. It would suck if for each alias token we had to look inside and check if we need to drop the elevated color per usage.

Reply:
Reading Apple's documentation, the elevated variant seems to be precisely for things like popovers and sheets. If whatever surface presents the popover is BG4, and the popover is also BG4, in dark mode the elevated variant will ensure the popover doesn't visually clash with the presenting surface. Isn't that what we want?

Reply:
It's mostly the intended behavior with a few exceptions. For instance, for DateTimePicker design requested specifically we don't use darkElevated colors. And here, the figma of the popover determines the dark variant to be used for dark mode.

Reply:
Can we verify with design whether this is the expected fallback? It's possible that the component design predates the existence of dark elevated, which is relatively newer for the toolkit.

Reply:
Confirmed with design this is the expected behavior for the popover's background color.